app/code/Magento/InventoryBundle/etc/module.xml

```diff
@@ -5,6 +5,11 @@
  * See COPYING.txt for license details.
  */
 -->
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_InventoryBundle" setup_version="1.0.0" />
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Magento_InventoryBundle" setup_version="1.0.0">
+        <sequence>
```

Comment:
We don't need to have sequence on `Magento_InventoryBundle`

Reply:
@naydav fixed.